src/modules/tests/category/category.actions.ts

```diff
@@ -0,0 +1,11 @@
+import { Action } from '@ngrx/store';
+
+export const POPULATE_TEST_CATEGORY = '[Waiting Room] populating test category';
```

Comment:
Needs a rename as it's not being used on the waiting room page any more

Reply:
Done